welcome.md

```diff
@@ -19,6 +19,127 @@ Here is what sets it apart:
 * **Batteries included**: Includes probability distributions, Gaussian processes, ABC, SMC and much more. It integrates nicely with {doc}`ArviZ <arviz:index>` for visualizations and diagnostics, as well as {doc}`Bambi <bambi:index>` for high-level mixed-effect models.
 * **Community focused**: Ask questions on [discourse](https://discourse.pymc.io), join [MeetUp events](https://meetup.com/pymc-online-meetup/), follow us on [Twitter](https://twitter.com/pymc_devs), and start [contributing](https://www.pymc.io/projects/docs/en/latest/contributing/index.html).
 
+## Example from Linear Regression
+
+This example demonstrates how to perform Bayesian inference for a linear regression model to predict plant growth based on environmental factors.
+
+Plant growth can be influenced by multiple factors, and understanding these relationships is crucial for optimizing agricultural practices.
+
+```python
+import pymc as pm
+
+# Taking draws from a normal distribution
+seed = 42
+x_dist = pm.Normal.dist(shape=(100, 3))
+x_data = pm.draw(x_dist, random_seed=seed)
+
+# Independent Variables:
```

Comment:
I would try to minimize code comments to specific annotations that need to go right beside the code itself. For more textual elements like this I would split the code in two and use markdown